### DISCUSSION TITLE
X-for is not listing all the items

### DISCUSSION THREAD

**Discussion by blpraveen** [alpinejs/alpine#3651] [7. Bugs (archived)]
With V3 alpinejs unable to list the user items

Please see the issue recreated in Codepen

[V3](https://codepen.io/B-L-Praveen/pen/zYMzZVv)

In older version 
it is working fine 

[V1.9.4](https://codepen.io/B-L-Praveen/pen/WNYOpzK)

**Comment by ekwoka** [comment]
the codepen doesn't work at all. Please resolve the console errors

**Reply by blpraveen** [reply]
Why I get errors?. I am not getting same error in old version.

**Reply by ekwoka** [reply]
probably something in the way codepen handles the JS and Alpine initializes itself.

codepen runs the attached js in their system (alpine in this case) before it runs your personal js.

So you should instead import it as a module from, as an example, esm.sh, have your code, and then start alpine.

Other issues you have: x-if and x-for can only have one child, not multiple.

Don't even need the x-if at all

**Reply by blpraveen** [reply]
Okay I recreate this issue esm.sh,?

**Reply by blpraveen** [reply]
It is working fine in esm.sh

**Reply by ekwoka** [reply]
Yeah, then it's working fine.

The issue before was just codepen. Not alpine or your code.
